docs/next/guides/building-and-testing/building.md

```diff
@@ -9,83 +9,263 @@ tags:
+
+To build the React package from the root directory:
+1. Make sure you meet the [building requirements](#building-requirements).
+2. Go to the root directory.
+3. Run `npm run in react build`.<br>Only the React package builds.
+
+#### React build tasks
+
+You can also run individual React `build` tasks:
+
+::: details React build tasks
+`build:commonjs`
+  - Transpiles files into the `CommonJS` format.
+  - During release, these files are published into the npm repository.
+
+`build:umd`
+  - Creates the following bundles compatible with the Universal Module Definition:
+    - `dist/handsontable.js`
+    - `dist/handsontable.css`
+    - `dist/handsontable.full.js`
+    - `dist/handsontable.full.css`
+
+`build:es`
+  - Transpiles files into the `import/export` format.
+  - During release, these files are published into the npm repository.
+
+`build:min`
+  - Creates the minified bundles:
+    - `dist/handsontable.min.js`
+    - `dist/handsontable.min.css`
+    - `dist/handsontable.min.full.js`
+    - `dist/handsontable.min.full.css`
+:::
+
+### Building the Vue package
+
+To build the Vue package:
+1. Make sure you meet the [building requirements](#building-requirements).
+2. Go to `./wrappers/vue`.
+3. Run `npm run build`.<br>Only the Vue package builds.
+
+To build the Vue package from the root directory:
+1. Make sure you meet the [building requirements](#building-requirements).
+2. Go to the root directory.
+3. Run `npm run in vue build`.<br>Only the Vue package builds.
+
+#### Vue build tasks
+
+You can also run individual Vue `build` tasks:
+
+::: details Vue build tasks
+`build:commonjs`
+  - Transpiles files into the `CommonJS` format.
+  - During release, these files are published into the npm repository.
+
+`build:umd`
+  - Creates the following bundles compatible with the Universal Module Definition:
+    - `dist/handsontable.js`
+    - `dist/handsontable.css`
+    - `dist/handsontable.full.js`
+    - `dist/handsontable.full.css`
+
+`build:es`
+  - Transpiles files into the `import/export` format.
+  - During release, these files are published into the npm repository.
 
-To run your own build, follow the below steps:
+`build:min`
+  - Creates the minified bundles:
+    - `dist/handsontable.min.js`
+    - `dist/handsontable.min.css`
+    - `dist/handsontable.min.full.js`
+    - `dist/handsontable.min.full.css`
 
-1. Install Node.js (available for Windows, macOS, and Linux). This will also install NPM - Node Package Manager, which handles all the dependencies. Handsontable requires **Node.js >=15.11.0** and npm **>=7.1.1** for building and testing.
-2. Clone the Handsontable repository on your local disk and go to the directory where you cloned it.
-3. Run `npm install` to download all the dependencies defined in `package.json`. The dependencies will be downloaded into a new directory `node_modules`, which is ignored by Git.
-4. **Run `npm run build` to make your first build!**
+`build:esDts`
```

Comment:
Probably not useful to contributors, I'd skip that.

Reply:
Done: b635371